contact.html

```diff
@@ -17,3 +17,4 @@ <h1>Let's Talk!</h1>
     <input type="submit" value="Send">
     <br>
 </form>
+<p>Got a secret?<a href="/.well-known/pgpKey.txt" target="_blank"> Encrypt using my PGP key!</a> Remember to attach/link to your own PGP key, so that we are abe to reply securely.</p>
```

Comment:
```suggestion
<p>Got a secret?<a href="/.well-known/pgpKey.txt" target="_blank"> Encrypt using my PGP key!</a> Remember to attach/link to your own PGP key, so that we are able to reply securely.</p>
```